Model/Config/Installment.php

```diff
@@ -9,4 +9,29 @@ class Installment extends Config
      * @var string
```

Comment:
Docblock should be `@const`? Saw [this](https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-javascript.html#frontend-dev-guide-javascript-js-coding-docblock-tagreference)

Reply:
my code editor does not like that.. but ok